addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1768,8 +1775,8 @@ export class Wysiwyg extends Component {
      * @returns {Promise}
      */
     waitForEmptyMutexAction() {
-        if (this.snippetsMenu) {
-            return this.snippetsMenu.execWithLoadingEffect(() => null, false);
+        if (this.state.showSnippetsMenu) {
+            return this.mutex.getUnlockedDef();
```

Comment:
Did you adapt this to still have an effect when `waitForEmptyMutexAction` is called?